### DISCUSSION TITLE
creating new instance of contract

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/foundry-full-course-cu#932] [Q&A]
```
MyContract myContract = new MyContract();
```
AND
```
new MyContract();
```
Whats mainly the difference between them?

**Comment by Niferu** [comment]
Hi @cell0901 

The main difference is that in upper lane you can track newly created/deployed contract by calling `myContract` and in lower lane you are just creating new contract and cannot call it at all without getting it somehow later as you are not saving it under variable type MyContract like you do up.
